test/manipulation.test.js

```diff
@@ -1338,6 +1338,25 @@ describe('manipulation', function() {
         });
       }
     });
+    describe('replaceById', function() {
```

Comment:
I think you need to skip this block when the connector does not implement `replaceById`:

```js
var connectorHasReplaceById = !!getSchema().connector.replaceById;
bdd.describeIf(connectorHasReplaceById, 'replaceById', function() {
  // ...
});
```

See [test/kvao/delete.suite.js](https://github.com/strongloop/loopback-datasource-juggler/blob/736a4ad1856077c23a053a14165a3707799abd77/test/kvao/delete.suite.js) for an example how to setup and use `bdd.describeIf` and [test/manipulation.test.js#L885](https://github.com/strongloop/loopback-datasource-juggler/blob/c9283cffdaf76f79e5611270f9777fd88056271d/test/manipulation.test.js#L885) for an existing example detecting `replaceId` availability.